pkg/minishift/addon/parser/addon_parser.go

```diff
@@ -182,10 +182,17 @@ func (parser *AddOnParser) parseCommands(scanner *bufio.Scanner) ([]command.Comm
 func createMetaData(header []string) (addon.AddOnMeta, error) {
 	regex, _ := regexp.Compile(`^# ?(.*):(.*)`)
 	metaMap := make(map[string]string)
+	var key string
 	for _, line := range header {
 		matches := regex.FindAllStringSubmatch(line, -1)
+		if len(matches) == 0 {
```

Comment:
So this is when there is no match, right? In this case you append to 'metaMap[key]', right? What if metaMap[key] is not set yet? What if someone does:

    # Hello 
    # Name:foo

Would you not get a panic there. Personally, I probably would aggregate multiple lines in a separate variable and only add things to the metaMap once I progress to the next tag or each the end of the header. Might also be worth to extract this into a function with a name describing what this is about, e.g. _handleMultileLine_. Took me a while to understand the code,